src/pages/editAccount/main.tsx

```diff
@@ -0,0 +1,92 @@
+import { useRef, useState } from 'react';
+// import Input from '../../components/input/input';
+import Input from '../../components/input/input';
+import Button from '../../components/button/button';
+import { checkValues, handler } from './hooks/hooks';
+import styles from './main.module.scss';
+import { useNavigate } from 'react-router-dom';
+
+const Changepassword = () => {
+  const [isPending, setisPending] = useState(false);
+  const oldPass = useRef<HTMLInputElement | null>(null);
+  const newPass = useRef<HTMLInputElement | null>(null);
+  const confirmPass = useRef<HTMLInputElement | null>(null);
+  const navigate = useNavigate();
+
+  async function handle() {
+    setisPending(true);
+    const divElement = document.getElementsByClassName('change-password-error')[0];
```

Comment:
this is not a good approach to work within React TSX, you have a virtual document already. use Redux or useState to get the value and put them inside the TSX elements there down., what if a class is changed?